subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ResolvedFilesCollectingVisitor.java

```diff
@@ -70,11 +74,7 @@ public void visitFile(ComponentArtifactIdentifier artifactIdentifier, AttributeC
     private void addArtifactFiles() {
         if (!artifacts.isEmpty()) {
             for (ResolvedArtifact artifact : artifacts) {
-                try {
-                    this.files.add(artifact.getFile());
-                } catch (Throwable t) {
-                    failures.add(t);
-                }
+                this.files.add(artifact.getFile());
```

Comment:
I think `artifacts` might have been used, when this was originally implemented, but it changed after a rebase. 

So I agree. Changed it.